### ISSUE TITLE
Add Sina Weibo and WeChat Share Button 

### ISSUE THREAD

**Issue by shirleyDDD** [bitshares/bitshares.org#103]
Sina Weibo and Wechat are the most popular online social media in China. Adding Sina Weibo and WeChat Share Button on the news.bitsahres.org website will be a great way to share and deliver the content of BitShares to the Chinese community.

I write this issue after having a conversation with Digital Lucife about this suggestion. The news.bitsahres.org will publish the multilanguage version, which means the BitShares news will be readable for Chinese. However, we also need to find more ways to spread the news.

**Comment by ryanRfox** [comment]
Yes, good plan. Tracking this with an Issue is best as you noted.

Best,
Ryan
________________________________
